### PR TITLE
Bump versions and update changelog for v1.0.3 release

### PR DESCRIPTION
**Pull request by adamwg**

### CODE REVIEW

File: deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+            - "--v=5"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: "Always"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: csi-attacher
+          image: quay.io/k8scsi/csi-attacher:v1.0.3
```
Comment (waynr) [inline deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml]: I don't see this version in the repository: https://quay.io/repository/k8scsi/csi-attacher?tab=tags

File: deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: "Always"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: csi-snapshotter
+          image: quay.io/k8scsi/csi-snapshotter:v1.0.3
```
Comment (waynr) [inline deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml]: I don't see this version in the repository: https://quay.io/repository/k8scsi/csi-snapshotter?tab=tags

File: deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml [hunk 3]
```diff
@@ -0,0 +1,532 @@
...
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: Always
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: csi-cluster-driver-registrar
+          image: quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.3
```
Comment (waynr) [inline deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml]: I don't see this version in the repository: https://quay.io/repository/k8scsi/csi-cluster-driver-registrar?tab=tags

File: deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml [hunk 4]
```diff
@@ -0,0 +1,532 @@
...
+      labels:
+        app: csi-do-node
+        role: csi-do
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccount: csi-do-node-sa
+      hostNetwork: true
+      containers:
+        - name: csi-node-driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.3
```
Comment (waynr) [inline deploy/kubernetes/releases/csi-digitalocean-v1.0.3.yaml]: I don't see this version in the repository: https://quay.io/repository/k8scsi/csi-cluster-driver-registrar?tab=tags

**Comment by waynr** [comment]
Hmm, as a matter of fact given that the manifests are copied from the previous version and `sed`ed to replace the old version with the new version, it seems like there is a good chance that the `1.0.2` manifest is similarly incorrect and was never fixed because there was no issue raised for it.

**Comment by adamwg** [comment]
@waynr Good catch, fixed.

**Pull request merged**